third_party/move/move-prover/tests/testsuite.rs

```diff
@@ -168,6 +169,7 @@ fn test_runner_for_feature(path: &Path, feature: &Feature) -> datatest_stable::R
 
     let mut error_writer = Buffer::no_color();
     let result = if feature.v2 {
+        options.language_version = Some(LanguageVersion::V2_0);
```

Comment:
So why do we see the resource index error then?

Reply:
By adding this, error will only be generated for the newly added test case `third_party/move/move-prover/tests/sources/functional/resource_index.move`  when running prover with V1. No error is generated for V2.

Reply:
Update the code so that the error disappears.